files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml

```diff
@@ -1,105 +1,76 @@
+  ################################
+
+  interactive_pulsar:
+    inherits: basic_docker_destination
+    runner: pulsar_embedded
+    destination_name_override: "embedded_pulsar_interactive_c{cores}-m{mem}"
+    max_accepted_cores: 24
+    max_accepted_mem: 128
     scheduling:
-      prefer:
+      accept:
         - docker
+      require:
         - interactive
 
   interactive_pulsar_gpu:
     inherits: interactive_pulsar
     destination_name_override: "e-pulsar_i_c{cores}-m{mem}-g{gpus}"
     max_accepted_gpus: 1
-
     env:
-      #GALAXY_MEMORY_MB: '{MEMORY_MB}'
-      #GALAXY_SLOTS: '{PARALLELISATION}'
-      GPU_AVAILABLE: '1'
+      GPU_AVAILABLE: "1"
     params:
       requirements: 'GalaxyGroup == "compute_gpu"'
-      tmp_dir: true
-      metadata_strategy: 'extended'
+      metadata_strategy: "extended"
     scheduling:
-      prefer:
-        - docker
-        - interactive
+      require:
+        - gpu
 
   embedded_pulsar_docker:
+    inherits: basic_docker_destination
     runner: pulsar_embedded
     destination_name_override: "embedded_pulsar_docker_c{cores}-m{mem}"
-    params:
-      tmp_dir: true
-      docker_enabled: true
-      #docker_volumes: $defaults
-      docker_sudo: false
-      docker_net: bridge
-      docker_auto_rm: true
-      docker_set_user: ""
-      require_container: true
-      submit_request_cpus: "{cores}"
-      submit_request_memory: "{mem}"
-      outputs_to_working_directory: false
-      container_monitor_result: callback
-      submit_requirements: 'GalaxyDockerHack == True'
     scheduling:
       prefer:
         - docker
         - embedded-pulsar
 
-  condor_singularity:
-    runner: condor
-    max_accepted_cores: 24
-    max_accepted_mem: 128
-    params:
-      singularity_enabled: true
-      singularity_volumes: "$_CONDOR_SCRATCH_DIR:rw,$job_directory:rw,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/data/db/:ro,/data/dnb01/galaxy_db/:ro,/data/dnb02/galaxy_db/:ro,/data/dnb03/galaxy_db/:ro,/data/dnb05/galaxy_db/:ro,/data/dnb06/galaxy_db/:rw,/data/dnb07/galaxy_db/:rw,/data/dnb08/galaxy_db/:rw,/data/dp01/galaxy_db/:rw,/data/0/galaxy_db/:ro,/data/1/galaxy_db/:ro,/data/2/galaxy_db/:ro,/data/3/galaxy_db/:ro,/data/4/galaxy_db/:ro,/data/5/galaxy_import/galaxy_user_data/:ro,/data/6/galaxy_db/:ro,/data/7/galaxy_db/:ro,/usr/local/tools/:ro"
-      singularity_default_container_id: "/cvmfs/singularity.galaxyproject.org/all/centos:8.3.2011"
-      tmp_dir: true
-      requirements: 'GalaxyGroup == "compute"'
-      metadata_strategy: "extended"
-    scheduling:
-      prefer:
-        - singularity
-
-  condor_tpv:
-    runner: condor
-    max_accepted_cores: 64
-    max_accepted_mem: 1000
-    scheduling:
-      accept:
-        - condor-tpv
-    params:
-      tmp_dir: true
-
-  condor_extended:
-    inherits: condor-tpv
-    params:
-      metadata_strategy: 'extended'
-    scheduling:
-      require:
-        - extended-metadata
+  #######################
```

Comment:
would it help or only confuse if we split those up into separate files?

Reply:
I would try to stick to few files